### PR TITLE
Formatting: convert indent from visual to hanging

### PR DESCRIPTION
**Pull request by Glutexo**
Covert all visual indentation to hanging, preferred by [Black](https://github.com/python/black). Some lines were transformed to have an optimal length. Lines that violates [Black’s](https://github.com/python/black) standards only in terms of length, but don’t in terms of the indentation were not touched. Split multiline strings have been recompiled to have the longest lines possible.

This is a part towards a cleanly formatted code that can be transformed by [Black](https://github.com/python/black) on every commit. #335

### CODE REVIEW

**Comment by Glutexo** [comment]
This pull request has been approved. Can we merge it?

**Comment by Glutexo** [comment]
Rebased, mergeable again.

**Comment by Glutexo** [comment]
Rebased to resolve conflicts.

**Comment by Glutexo** [comment]
And one more conflict-resolving rebase.

**Comment by dehort** [comment]
I manually resolved the conflict and pushed this as commit 257e0de2d6934af8882d4638e711de1b77cbb8e6.

File: test_api.py [hunk 1]
```diff
@@ -1181,12 +1185,13 @@ def test_patch_on_multiple_hosts_with_some_non_existent(self):
...
         original_id = self.added_hosts[0].id
 
-        invalid_data_list = [{"ansible_host": "a"*256},
-                             {"ansible_host": None},
-                             {},
-                             {"display_name": None},
-                             {"display_name": ""},
-                             ]
+        invalid_data_list = [
+            {"ansible_host": "a"*256},
```
Comment (Glutexo) [inline test_api.py]: Whitespace fix has not survived the merge.

File: test_api.py [hunk 2]
```diff
@@ -937,9 +942,9 @@ def test_create_host_with_system_profile_with_invalid_data(self):
         host["rhel_machine_id"] = generate_uuid()
 
         # List of tuples (system profile change, expected system profile)
-        system_profiles = [{"infrastructure_type": "i" * 101,
-                            "infrastructure_vendor": "i" * 101,
-                            "cloud_provider": "i" * 101}]
+        system_profiles = [
+            {"infrastructure_type": "i"*101, "infrastructure_vendor": "i"*101, "cloud_provider": "i"*101}
```
Comment (Glutexo) [inline test_api.py]: Whitespace fixes have not survived the merge.

**Pull request closed without merging**